server/.env

```diff
@@ -0,0 +1,20 @@
+# Script docker run Postgres
```

Comment:
Why do you have this file commented entirely here?

Reply:
upss, just deleted.